test/util/bitcoin-util-test.py

```diff
@@ -63,87 +66,93 @@ def bctester(testDir, input_basename, buildenv):
     else:
         sys.exit(0)
 
-def bctest(testDir, testObj, buildenv):
+
+def bctest(test_dir, test_obj, buildenv):
     """Runs a single test, comparing output and RC to expected output and RC.
 
     Raises an error if input can't be read, executable fails, or output/RC
     are not as expected. Error is caught by bctester() and reported.
     """
     # Get the exec names and arguments
-    execprog = buildenv["BUILDDIR"] + "/src/" + testObj['exec'] + buildenv["EXEEXT"]
-    execargs = testObj['args']
+    execprog = buildenv["BUILDDIR"] + "/src/" + test_obj['exec'] + buildenv["EXEEXT"]
```

Comment:
I'd assume those should probably use `os.path.join` to make it work on other os as well.